pkg/security/jwt/cache.go

```diff
@@ -0,0 +1,70 @@
+package jwt
+
+import (
+	"time"
+
+	"github.com/plgd-dev/go-coap/v3/pkg/cache"
+	pkgTime "github.com/plgd-dev/hub/v2/pkg/time"
+)
+
+type TokenRecord struct {
+	Blacklisted bool
+}
+
+type TokenCache struct {
+	cache *cache.Cache[string, TokenRecord]
+}
+
+func NewTokenCache() *TokenCache {
+	return &TokenCache{
+		cache: cache.NewCache[string, TokenRecord](),
+	}
+}
+
+func getCacheKey(tokenID, issuer string) string {
+	return tokenID + "." + issuer
```

Comment:
this could be very big keys ~ 69bytes 37(uuid) + ~32 (https://{host}/m2m-oauth-server)